package.json

```diff
@@ -31,7 +31,7 @@
   "devDependencies": {
     "@antora/cli": "^2.1.2",
     "@antora/site-generator-default": "^2.1.2",
-    "@typechain/ethers-v5": "^7.0.1",
+    "@typechain/ethers-v5": "^7.1.0",
```

Comment:
Update typechain to be able to pull contracts events from generated contract types.